src/script.js

```diff
@@ -288,14 +288,23 @@ function getResponse(query) {
             composeSusiMessage(response);
         },
         success: function (data) {
-            data.answers[0].actions.map((action) => {
-            var response = composeResponse(action,data.answers[0].data);
-            loading(false);
-            composeSusiMessage(response);
-            if(action.type !== data.answers[0].actions[data.answers[0].actions.length-1].type){
-                loading(); //if not last action then create another loading box for susi response
-            }
-            });
+        	if (data.answers && data.answers[0]) {
+        		data.answers[0].actions.map((action) => {
+		            var response = composeResponse(action,data.answers[0].data);
+		            loading(false);
+		            composeSusiMessage(response);
+		            if(action.type !== data.answers[0].actions[data.answers[0].actions.length-1].type){
+		                loading(); //if not last action then create another loading box for susi response
+		            }
+		        });
+        	} else {
+        		var response = composeResponse({
+        			type: "answer",
+        			expression: "SUSI could not find an answer to your question."
```

Comment:
Can we keep it : `No response found`
As this one is a bit long.

Reply:
And it is not necessary that the input is a question. It can be a general statement that maybe susi can produce no response of.